library/std/src/sys/thread_local/exit/unix.rs

```diff
@@ -0,0 +1,10 @@
+use crate::mem;
+
+pub unsafe fn at_process_exit(cb: unsafe extern "C" fn()) {
+    // Miri does not support atexit.
```

Comment:
Could you file an issue for this in the Miri repo? If std starts using this, we should consider supporting it.